src/commands/version.js

```diff
@@ -0,0 +1,275 @@
+const dirsExist = require('../utils/dirs-exist');
+const { terminal } = require('terminal-kit');
+const fs = require('fs');
+const path = require('path'); // Path module
+const semver = require('semver');
+
+exports.command = 'version [type] [projects]';
+exports.desc = 'Run an npm version bump process.';
+exports.builder = (yargs) => {
+  yargs.option('type', {
+    describe: 'type of version to bump.',
+    default: 'string',
```

Comment:
`string` is not a valid default for this. What yargs is expecting here is the default value for when something does not exist. So in this case, running the command without the type is the same as running it the running it with `string`.

**Example**
```
build-tools version
```
is the same as
```
build-tools version string
```

There is also an additional side effect to this. If running the command without a `major`, `minor` or `patch`, the version will be changed to `null` across all applicable files. A default of `patch` would likely be the most appropriate action here for when these are not explicitly set. Ideally however, incrementing the lowest defined value should be done. So if we have `1.1.0-beta.1` and the command is run, `1.0.0-beta.2` should be the next version incremented.

**Output**
```
Error incrementing version in client-mu-plugins/test-client-plugin/package.json: Invalid version: undefined
Error incrementing version in plugins/standards/package.json: Invalid version: undefined
Error incrementing version in plugins/test-plugin/package.json: Invalid version: null
#themes/test-theme/package.json: 1.8.0 -> null
#themes/test-theme/style.css: 1.2.0 -> null
```

Reply:
I have changed this now so it increments the lowest value http://bigbite.im/i/D3gdMt